src/components/assetlib_projects/asset_library_projects.tscn

```diff
@@ -0,0 +1,223 @@
+
+[node name="ScrollContainer" type="ScrollContainer" parent="MarginContainer"]
+layout_mode = 2
+size_flags_vertical = 3
+
+[node name="VBoxContainer" type="VBoxContainer" parent="MarginContainer/ScrollContainer"]
+layout_mode = 2
+size_flags_horizontal = 3
+
+[node name="NavigationButtons" type="HBoxContainer" parent="MarginContainer/ScrollContainer/VBoxContainer"]
+unique_name_in_owner = true
+visible = false
+layout_mode = 2
+alignment = 1
+
+[node name="FirstButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+text = "First"
+
+[node name="PreviousButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+text = "Previous"
+
+[node name="VSeparator" type="VSeparator" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+
+[node name="PageButtons" type="HBoxContainer" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+
+[node name="VSeparator2" type="VSeparator" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+
+[node name="NextButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+text = "Next"
+
+[node name="LastButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+text = "Last"
+
+[node name="AssetList" type="HFlowContainer" parent="MarginContainer/ScrollContainer/VBoxContainer"]
+unique_name_in_owner = true
+layout_mode = 2
+
+[node name="NavigationButtons2" type="HBoxContainer" parent="MarginContainer/ScrollContainer/VBoxContainer"]
+visible = false
+layout_mode = 2
+size_flags_vertical = 8
+alignment = 1
+
+[node name="FirstButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons2"]
+layout_mode = 2
+text = "First"
+
+[node name="PreviousButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons2"]
+layout_mode = 2
+text = "Previous"
+
+[node name="VSeparator" type="VSeparator" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons2"]
+layout_mode = 2
+
+[node name="PageButtons" type="HBoxContainer" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons2"]
+layout_mode = 2
+
+[node name="VSeparator2" type="VSeparator" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons2"]
+layout_mode = 2
+
+[node name="NextButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons2"]
+layout_mode = 2
+text = "Next"
+
+[node name="LastButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons2"]
+layout_mode = 2
+text = "Last"
+
+[node name="ColorRect" type="ColorRect" parent="MarginContainer"]
+unique_name_in_owner = true
+layout_mode = 2
+color = Color(0.211765, 0.239216, 0.290196, 0.694118)
```

Comment:
should be removed since it doesn't look good with other themes (Light for instance)
My proposal is removing the ColorRect and replacing it with the logic Godot's Project Manager uses: just dim the ScrollContainer (Color(1, 1, 1, 0.5). the cheapest way to preserve both the good appearance and multi-theming support.